### DISCUSSION TITLE
[Minor] duplicates in SpamDomainList.txt

### DISCUSSION THREAD

**Discussion by nob-rot** [ThioJoe/YT-Spam-Lists#177] [General]
193 is duplicated at 199 
217 is duplicated at 225

something of nothing but there it is.

**Comment by Firecul** [comment]
ok, one of them was me 😥

(there are more duplicates than that btw, 3 iirc, all of them were there before I started 😉 )

**Reply by Firecul** [reply]
I'll fix these ones with my next push though, and 1 more I caused in the video ids you missed lol
